src/stores/appStore.ts

```diff
@@ -0,0 +1,110 @@
+import { defineStore } from "pinia";
+import { Feature } from "geojson";
+import { getAppConfig, getGeojson, getIndicators } from "../resources/utils";
+import { AppConfig } from "../resourceTypes";
+import { FeatureGeojson, FeatureIndicators } from "../types";
+
+export const useAppStore = defineStore("app", {
+    state: () => {
+        const loading = true;
+        const selectedIndicator = "";
+
+        // When a country is selected, we display admin2 data for it.
+        // We display admin1 data for all other countries
+        const selectedCountryId = "";
+
+        const appConfig: AppConfig | null = null;
+
+        // We keep all data in dictionaries with country ids as keys, mirroring the resources on disk
+        const admin1Indicators: Dict<FeatureIndicators> = {};
+        const admin1Geojson: Dict<FeatureGeojson> = {};
+        const admin2Indicators: Dict<FeatureIndicators> = {};
+        const admin2Geojson: Dict<FeatureGeojson> = {};
+
+        return {
+            loading,
+            appConfig,
+            selectedCountryId,
+            selectedIndicator,
+            admin1Indicators,
+            admin1Geojson,
+            admin2Indicators,
+            admin2Geojson
+        };
+    },
+    getters: {
+        selectedIndicators: (state): FeatureIndicators => {
+            const { selectedCountryId, admin1Indicators, admin2Indicators } = state;
+            // get single dictionary of feature id to indicator values for user selections
+            if (!selectedCountryId) {
+                return Object.assign({}, ...Object.values(admin1Indicators));
```

Comment:
Unfortunately this doesn't quite get us what we want, because Object.values is an array - what you actually end up with is an object with keys corresponding to the array indexes (so effectively replacing the country keys with those indexes as keys) rather than flattening the whole dictionary down to feature level.